sig-multitenancy/concepts-and-designs/user-group-designs.md

```diff
@@ -0,0 +1,120 @@
+# User Group Design
+
+## Objective
+
+Kubesphere uses RBAC authorization to drive authorization decisions. But roles only can be assigned to individual users currently, which is inconcinnity in some scenarios. For example, one team may need to manage multiple projects in a workspace, but we can only grant all projects permission with the role of workspace level, or we have to invite users to those projects one by one and grant with a role of project level.
+
+By binding the project role to a Group of workspace, we can grant multiple projects' permission to the group then all the users in the group have access to those projects.
+
+### User group management of platform level
+
+At the platform level, the user group will be shown as a hierarchical structure of workspace and groups of the workspace. Admins will be able to assign users to a group.
+
```

Comment:
It is better to draw a picture here since people is easy to get confused by groups of the workspace